docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +137,94 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
 - You can choose dedicated IOPs EBS drives on AWS and specify the number of dedicated IOPS you need.
 
 YugabyteDB Anywhere also allows creating these machines out of band and importing them as an on-premises install.
+
+## Node agent
+
+### What is a node agent?
+
+Node agent is an RPC service running on a YugabyteDB node, and is used to manage communication between YugabyteDB Anywhere and the nodes in universes. It includes the following functions:
+
+- Invoke shell commands directly on the remote host, similar to running commands over SSH. Like SSH, the agent also does shell-login such that any previous command modifying the environment in the resource files (for example, `~/.bashrc`) gets reflected in the subsequent command.
+- Invoke procedures or methods on the node agent.
+- Additionally, for on-premises (manually provisioned nodes) deployments, node agent also functions as a utility to run preflight checks, and add node instances.
+
+### How is node agent installed on a YugabyteDB node?
+
```

Comment:
I know that this is redundant information but it might be good to add this information here about the ways YBA allows users to provision the node (@aishwarya24 and @nkhogen please see if this is redundant)

Currently there are 3 ways of provisioning the node depending upon the level of access provided to YBA:

- First, is automatic provisioning where an ssh user with sudo access is given to YBA for the node (e.g. the ec2-user for AWS EC2 instances)
- Second, is assisted manual provisioning via a script (provision_instance.py) where YBA doesn't have access to an ssh user with sudo access BUT the user can run a script interactively on YBA and provide the script with parameters for credentials for the ssh user with sudo access
- Third, is fully manual provisioning where neither YBA has access to an ssh user with sudo access nor the user has access to an ssh user with sudo access. In this case, only a local (non-ssh) user is available with sudo access and the user has to follow manual steps (link to fully manual provisioning) to provision the node